client/src/main/scala/drt/client/components/ContactPage.scala

```diff
@@ -146,7 +146,7 @@ object FeedBackComponent {
               val aORbTest = abFeatures.headOption.map(_.abVersion).getOrElse("B")
               val bannerHead = aORbTest match {
                 case "A" => "Your feedback improves DRT for everyone"
-                case _ => "Help us improve your DRT experience"
+                case _ => "Help us improve DRT experience"
```

Comment:
This doesn't read right to me. There's a missing pronoun - "Help us improve the DRT experience"